code/__DEFINES/atmospherics_defines.dm

```diff
@@ -156,6 +156,7 @@
 #define ATMOS_MODE_SPACE 0
 #define ATMOS_MODE_SEALED 1
 #define ATMOS_MODE_EXPOSED_TO_ENVIRONMENT 2
+#define ATMOS_MODE_NO_DECAY 3
```

Comment:
Could you add a comment here to indicate what this means on the DM side?

Reply:
done!